File: Android/_setup/_ProjectName_/src/main/java/com/twist/dataengine/entities/TM_FilterAttributeOption.java
package com.twist.dataengine.entities;

/**
 * Created by Twist Mobile on 4/16/2016.
 */
public class TM_FilterAttributeOption {
    public String name;
    public String taxo;
    public String slug;

    public TM_FilterAttributeOption() {

    }

    public TM_FilterAttributeOption(TM_FilterAttributeOption other) {
        this.name = other.name;
        this.slug = other.slug;
        this.taxo = other.taxo;
    }
}

File: IOS/TMStore/TMStore/Classes/Models/TM_FilterAttributeOption.m
//
//  TM_FilterAttributeOption.m
//  TMStore
//
//  Created by Rishabh Jain on 16/05/16.
//  Copyright © 2016 Twist Mobile. All rights reserved.
//

#import "TM_FilterAttributeOption.h"

@implementation TM_FilterAttributeOption
- (id)init {
    self = [super init];
    if (self) {
        self.name = @"";
        self.slug = @"";
        self.taxo = @"";
        self.isVisible = true;
    }
    return self;
}
- (id)init:(TM_FilterAttributeOption*)other {
    self = [super init];
    if (self) {
        self.name = other.name;
        self.slug = other.slug;
        self.taxo = other.taxo;
    }
    return self;
}
@end
